numbat-cli/src/main.rs

```diff
@@ -141,6 +156,17 @@ impl Cli {
         let load_prelude = !self.args.no_prelude;
         let load_init = !(self.args.no_prelude || self.args.no_init);
 
+        let user_config_path = Self::get_config_path().join("config.toml");
+
+        let config = match toml::from_str::<Config>(
+            &fs::read_to_string(&user_config_path).unwrap_or("".to_string()),
```

Comment:
Is there a better way to initialize an "empty"/default config than parsing an empty string?

Reply:
Yes. Let's implement `Default` for `Config` (not manually, but ideally by deriving it). And then we use `Config::default()` if no config file has been found.